app.test.js

```diff
@@ -101,14 +101,62 @@ describe('Server', () => {
       expect(revisedProject).toEqual(expectedResult);
     });
 
-    it('should return a sad 404 code if the targeted project is not found', async () => {
-      const invalidTargetId = -24;
+    // it('should return a sad 404 code if the targeted project is not found', async () => {
+    //   const invalidTargetId = -24;
+    //
+    //   const response = await request(app).get(`/api/v1/palettes/${invalidTargetId}`);
```

Comment:
I think the solution is adding `.send({invalidTargetId}`. to the end of the response line

Reply:
That worked!

Reply:
I've pushed up the tweaks. Way to go, man

Reply:
Wow